README.md

```diff
@@ -70,36 +71,90 @@ Run the installer. This will create the Devstack cluster, install and activate R
 ./romana-setup
 ```
 
+See details below about [using the system](#using-the-system) after the installation has completed.
+And [contact us](#getting-help) for more information or assistance.
+
+*Note:* By default, the stack name will be your username.
+If you prefer to use a different name for the stack, you can override this by providing an extra option when launching:
+```bash
+./romana-setup install -e stack_name=xyz
+```
+The name should be a single word, and only contain letters and numbers. (No hyphens, underscores, etc.)
+
 The EC2 installation takes 20-25 mins to complete (on t2.large instances) and creates an OpenStack DevStack cluster (Liberty release) with a single Controller Node and up to 4 Compute Nodes. Each OpenStack Node runs on a dedicated EC2 instance. Romana installs its OpenStack ML2 and IPAM drivers and creates a Romana router gateway interface on each Compute Node.
 
-Once installed, you can perform a variety of checks and experiments on your own. See the [Wiki](https://github.com/romana/romana/wiki) for details.
+
+### Teardown
 
 To uninstall, use the 'uninstall' subcommand.
 ```bash
 ./romana-setup uninstall
```

Comment:
Does this actually destroy the stack? Or the Vagrant VMs if this is for Vagrant?